modules/e2ee/OlmAdapter.js

```diff
@@ -207,6 +230,27 @@ export class OlmAdapter extends Listenable {
         }
     }
 
+    /**
+     * Sends sacMac if channel verification waas successful.
+     *
+     */
+    markParticipantVerified(participant, isVerified) {
+        const olmData = this._getParticipantOlmData(participant);
+
+        if (isVerified) {
+            const { sas, sasMacSent } = olmData.sasVerification;
+
+            if (sas && sas.is_their_key_set() && !sasMacSent) {
+                this._sendSasMac(participant);
+
+                // Mark the MAC as sent so we don't send it multiple times.
+                olmData.sasVerification.sasMacSent = true;
+            }
+        } else {
+            olmData.sasVerification = undefined;
```

Comment:
When would we create this object again? Also, we should likely log an error / warning, no?